src/components/textbox/__spec__.js

```diff
@@ -37,7 +37,7 @@ describe('Textbox', () => {
     });
 
     it('is decorated with a validation if a error is present', () => {
-      instance.setState({errorMessage: 'Error', valid: false});
+      instance.setState({errorMessage: 'Error', valid: false, messageShown: true});
```

Comment:
❓ Why do we need to set it explicitly now as state, compared to how it worked before?